Source/ZoomNet/Resources/Webinars.cs

```diff
@@ -162,6 +203,55 @@ public Task<RecurringWebinar> CreateRecurringWebinarAsync(string userId, string
 				.AsObject<RecurringWebinar>();
 		}
 
+		/// <summary>
+		/// Updates an existing recurring webinar for a user.
+		/// </summary>
+		/// <param name="webinarId">The webinar ID.</param>
+		/// <param name="topic">Webinar topic.</param>
+		/// <param name="agenda">Webinar description.</param>
+		/// <param name="start">Webinar start time.</param>
+		/// <param name="duration">Webinar duration (minutes).</param>
+		/// <param name="recurrence">Recurrence information.</param>
+		/// <param name="password">Password to join the webinar. By default the password may only contain the following characters: [a-z A-Z 0-9 @ - _ *]. Max of 10 characters. This can be updated within Zoom account settings.</param>
+		/// <param name="settings">Webinar settings.</param>
+		/// <param name="trackingFields">Tracking fields.</param>
+		/// <param name="cancellationToken">The cancellation token.</param>
+		/// <returns>
+		/// The async task.
+		/// </returns>
+		public async Task UpdateRecurringWebinarAsync(long webinarId, string topic = null, string agenda = null, DateTime? start = null, int? duration = null, RecurrenceInfo recurrence = null, string password = null, WebinarSettings settings = null, IDictionary<string, string> trackingFields = null, CancellationToken cancellationToken = default)
+		{
+			var data = new JObject();
+			data.AddPropertyIfValue("topic", topic);
+			data.AddPropertyIfValue("agenda", agenda);
+			data.AddPropertyIfValue("password", password);
+			data.AddPropertyIfValue("start_time", start?.ToUniversalTime().ToString("yyyy-MM-dd'T'HH:mm:ss'Z'"));
+			data.AddPropertyIfValue("duration", duration);
+			data.AddPropertyIfValue("recurrence", recurrence);
+			if (start.HasValue) data.Add("timezone", "UTC");
+			data.AddPropertyIfValue("settings", settings);
+			data.AddPropertyIfValue("tracking_fields", trackingFields?.Select(tf => new JObject() { { "field", tf.Key }, { "value", tf.Value } }));
+
+			var result = await _client
+				.PatchAsync($"webinars/{webinarId}")
+				.WithJsonBody(data)
+				.WithCancellationToken(cancellationToken)
+				.AsMessage()
+				.ConfigureAwait(false);
+
+			switch (result.StatusCode)
```

Comment:
I'm guessing this is debugging code that you didn't intend to commit?

Reply:
Damn it, give me 2 minutes...

Reply:
hahaha. No problem, like I said: I do this all the time!

Reply:
At the time I was pondering the possibility of throwing errors for the other possible error responses... Then I realised it was overkill ;)